package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ai",
+  "version": "0.1.0",
+  "description": "WordPress AI Experiments",
+  "license": "GPL-2.0-or-later",
+  "scripts": {
+    "plugin-zip": "wp-scripts plugin-zip",
+    "build": "echo 'Building...'",
```

Comment:
Adding these two scripts and installing `@wordpress/scripts` are the only changes here from what is currently in #38, just as an FYI for reviewers